scripts/redirects.js

```diff
@@ -0,0 +1,70 @@
+var fs = require('fs');
+var glob = require('glob');
+var utils = require('../lib/utils');
+
+var MASTER = 'master';
+
+hexo.extend.generator.register('blog-redirects', function () {
+  return expandRedirectObjs([
+    ['blog/2015/12/16/0.0.10-release/', 'blog/introducing-aframe/'],
+    ['blog/2015/12/16/introducing-aframe/', 'blog/introducing-aframe/'],
+    ['blog/2016/03/31/aframe-v0.2.0/', 'blog/aframe-v0.2.0/']
+  ]);
+});
+
+hexo.extend.generator.register('docs-redirects', function () {
+  var redirectObjs = [getDocRootRedirectObjs(), getPreVersionedRedirectObjs()];
+  redirectObjs.push([
+    ['docs/', 'docs/' + hexo.config.aframe_version + '/guide/'],
+    ['docs/guide/', 'docs/' + hexo.config.aframe_version + '/guide/'],
+    ['docs/components/loader.html', 'docs/components/collada-model.html'],
+    ['docs/core/animation.html', 'docs/core/animations.html'],
+    ['docs/core/assets.html', 'docs/core/asset-management-system.html'],
+    ['docs/core/mixin.html', 'docs/core/mixins.html'],
+    ['docs/core/templates.html', 'https://github.com/ngokevin/aframe-template-component'],
+    ['docs/guide/cameras-and-lights.html', 'docs/guide/building-a-basic-scene.html'],
+    ['docs/guide/entering-vr.html', 'http://mozvr.com/#start'],
```

Comment:
you don't think [http://localhost:4000/docs/0.1.0/primitives/a-box.html](http://localhost:4000/docs/0.1.0/primitives/a-box.html) should redirect to [http://localhost:4000/docs/0.1.0/primitives/a-cube.html](http://localhost:4000/docs/0.1.0/primitives/a-cube.html)?

I understand why - we could possibly do two redirects if a version isn't in the URL. seems fine.